web/packages/teleterm/src/ui/TopBar/Identity/IdentityList/IdentityListItem.tsx

```diff
@@ -15,85 +15,150 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+import { useCallback } from 'react';
+import styled from 'styled-components';
 
-import { useState } from 'react';
-
-import { ButtonIcon, Flex, Label, Text } from 'design';
+import { ButtonText, Flex, P3, Text } from 'design';
 import { Logout } from 'design/Icon';
+import { Cluster } from 'gen-proto-ts/teleport/lib/teleterm/v1/cluster_pb';
 
 import { useKeyboardArrowsNavigation } from 'teleterm/ui/components/KeyboardArrowsNavigation';
 import { ListItem } from 'teleterm/ui/components/ListItem';
 import { ProfileStatusError } from 'teleterm/ui/components/ProfileStatusError';
-import { getUserWithClusterName } from 'teleterm/ui/utils';
+import { useStoreSelector } from 'teleterm/ui/hooks/useStoreSelector';
+import { ProfileColor } from 'teleterm/ui/services/workspacesService';
 
-import { IdentityRootCluster } from '../useIdentity';
+import { UserIcon } from '../IdentitySelector/UserIcon';
 
 export function IdentityListItem(props: {
   index: number;
-  cluster: IdentityRootCluster;
+  cluster: Cluster;
   onSelect(): void;
-  onLogout(): void;
+  /** If defined, the logout button is rendered. */
+  onLogout?(): void;
 }) {
-  const [isHovered, setIsHovered] = useState(false);
   const { isActive } = useKeyboardArrowsNavigation({
     index: props.index,
     onRun: props.onSelect,
   });
-
-  const userWithClusterName = getUserWithClusterName(props.cluster);
+  const profileColor = useStoreSelector(
+    'workspacesService',
+    useCallback(
+      state => state.workspaces[props.cluster.uri]?.color,
+      [props.cluster.uri]
+    )
+  );
 
   return (
-    <ListItem
-      css={`
-        border-radius: 0;
-        height: auto;
-        min-height: 38px;
-      `}
+    <StyledListItem
       onClick={props.onSelect}
+      onKeyDown={e =>
+        (e.key === 'Enter' || e.key === 'Space') && props.onSelect()
+      }
       isActive={isActive}
-      onMouseEnter={() => {
-        setIsHovered(true);
-      }}
-      onMouseLeave={() => {
-        setIsHovered(false);
-      }}
+      title="Activate Cluster"
```

Comment:
`Switch to <cluster name>`? Maybe it's just me, but "Activate" has different connotations for me, you can activate your device for the first time etc.